hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/replicas/chunk/ChunkKeyHandler.java

```diff
@@ -190,9 +190,12 @@ protected void execute(OzoneClient client, OzoneAddress address)
             }
 
             if (isECKey) {
-              ChunkType blockChunksType = isECParityBlock(keyPipeline, entry.getKey()) ?
-                  ChunkType.PARITY : ChunkType.DATA;
-              jsonObj.put("chunkType", blockChunksType.name());
+              int replicaIndex = keyPipeline.getReplicaIndex(entry.getKey());
+              int dataCount = ((ECReplicationConfig) keyPipeline.getReplicationConfig()).getData();
+
+              ChunkType chunkType = (replicaIndex > dataCount) ? ChunkType.PARITY : ChunkType.DATA;
```

Comment:
Let's keep the earlier comment.
```suggestion
              // Index is 1-based,
              // e.g. for RS-3-2 we will have data indexes 1,2,3 and parity indexes 4,5
              ChunkType chunkType = (replicaIndex > dataCount) ? ChunkType.PARITY : ChunkType.DATA;
```

Reply:
Added. Thanks for review